codas/meta.yaml

```diff
@@ -16,9 +16,37 @@ requirements:
     run:
 
 
-#test:
-    #commands:
-        #- some_cmd --help ?
+test:
+    commands:
+        - atg 40.0 40.0 10000.0  # [not win]
+        - bvfreq 0 25 35 10 24.5 35 42  # [not win]
+        - depth 10000 30  # [not win]
+        - grav 10000.0 30.0  # [not win]
+        - press 0  # [not win]
+        - svan 35 12 1000.0  # [not win]
+        - svel 35 12 1000.0  # [not win]
+        - theta 40.0 40.0 10000.0 0.0  # [not win]
+        - to_date 1980 31  # [not win]
+        - to_day 2000 01 01 00 00  # [not win]
+        - to_week 1992 123.452  # [not win]
+        # FIXME: Should reflect conda re-locatable binary concept.
+        - codas_prefix  # [not win]
+        # FIXME: Segfaults!
+        - fix_bt_ss_bug  # [not win]
```

Comment:
@efiring I added a few CLI tests just to check if the paths are OK on Windows and I found out that but `fix_bt_ss_bug` is segfaulting on [Linux](https://circleci.com/gh/ioos/conda-recipes/1016), [OS X](https://travis-ci.org/ioos/conda-recipes/builds/108001648) and my local Windows VM (AppVeyor seems [fine](https://ci.appveyor.com/project/comtbot/conda-recipes/build/1.0.1866/job/egj23wj1l2gxu6d9)).

PS: I will not keep all these testing here in the final version of the package.